src/AccountProfileInline.jsx

```diff
@@ -31,6 +31,7 @@ const Wrapper = styled.a`
 `;
 
 const Text = styled.span`
+  display: inline-flex;
```

Comment:
This is going to break text truncation on overflow for really long account names. I had to remove `display: inline-flex;` here on a previous commit to fix it. What caused the need to add it back?

Reply:
This change was done by mistake after comparing with your's copy of `AccountProfileInline`